src/object/README.md

```diff
@@ -114,6 +114,39 @@ and any of the following sharding hints (percentage based on number of targets):
 - DAOS_OCH_SHD_EXT  - max(1024, 80%)
 - DAOS_OCH_SHD_MAX  - 100%
 
+### Object class naming conventions
```

Comment:
good explanations. however, the section right after this is "Data Protection Method" so there is some overlap and  breakage in the flow here.